examples/triton-docker-combined/terraform.tfvars.example

```diff
@@ -0,0 +1,7 @@
+account = "bobby.brown"
```

Comment:
I just discovered this too. The vars need to be defined before they can be initialized.

- `variables.tf` defines the variables.
- `terraform.tfvars` initializes them.